Security/src/CloudFoundrySingleSignon/README-UAA.md

```diff
@@ -0,0 +1,122 @@
+﻿# CloudFoundry Single Signon Security Sample App
+
+ASP.NET Core sample app illustrating how to make use of the Steeltoe [CloudFoundry External Security Provider](https://docs.steeltoe.io/api/v3/security/) for Authentication and Authorization against a CloudFoundry OAuth2 security service (e.g. [UAA Server](https://github.com/cloudfoundry/uaa) or [Tanzu Single Signon Service](https://docs.pivotal.io/p-identity/)).
+
+> NOTE: For simplicity, we've moved the [instructions for utilizing the SSO Tile](README.md) with this sample
+
+## Pre-requisites
+
+1. Cloud Foundry instance
+1. Windows support on Cloud Foundry (OPTIONAL)
+1. [.NET Core SDK](https://www.microsoft.com/net/download) installed
+1. [CloudFoundry UAA Command Line Client](https://github.com/cloudfoundry/cf-uaac) installed
+
+### Create OAuth2 Service Instance on CloudFoundry
+
+This sample requires an OAuth2 service in your org/space. These instructions will use the [UAA Server](https://github.com/cloudfoundry/uaa) as the OAuth2 service.
+
+### Target your Environment with UAA Command Line Tools
+
+Before creating the OAuth2 service instance, we need to use the UAA command line tool to establish some security credentials for our sample app. Target your UAA server:
+
+1. uaac target uaa.`YOUR-CLOUDFOUNDRY-SYSTEM-DOMAIN` (e.g. `uaac target uaa.system.testcloud.com`)
+
+Next, authenticate and obtain an access token for the `admin client` from the UAA server so that we can add our new application/user credentials. You will need the `Admin Client Secret` for your installation of CloudFoundry for this step. If you are using Tanzu Application Service (TAS), you can obtain the secret from the `Ops Manager/Elastic Runtime` credentials page under the `UAA` section.  Look for `Admin Client Credentials` and then use it as follows:
+
+1. uaac token client get admin -s `ADMIN_CLIENT_SECRET`
+
+> Note: To see the token that was retrieved, run the command `uaac contexts`
+
+### Add User and Group
+
+After authenticating, add a new `user` and `group` to the UAA Server database. Do NOT change the group name: `testgroup` as it is used for policy based authorization in the sample application. Feel free to change the username and password to anything you would like.
+
+1. uaac group add testgroup
+1. uaac user add dave --given_name Dave --family_name Tillman --emails dave@testcloud.com --password Password1!
```

Comment:
Perhaps replace 'dave' with something else? (multiple places in this file and others)